package.json

```diff
@@ -23,6 +24,9 @@
 		"bundle-analysis:run": "flub run bundleStats --dangerfile build-tools/packages/build-cli/lib/lib/dangerfile.js",
 		"changeset": "flub changeset add",
 		"check:versions": "flub check buildVersion -g client --path .",
+		"check:versions:fix": "pnpm run check:versions --fix",
+		"checks": "concurrently npm:policy-check npm:layer-check npm:syncpack:deps npm:syncpack:versions && npm run check:versions && npm run prettier",
+		"checks:fix": "npm run policy-check:fix && npm run syncpack:deps:fix && npm run syncpack:versions:fix && npm run check:versions:fix && npm run prettier:fix",
```

Comment:
This could look cleaner with [npm-run-all](https://github.com/bcomnes/npm-run-all2/blob/master/docs/npm-run-all.md) but probably not worth the extra dependency.